pkg/config/config_local.go

```diff
@@ -30,6 +30,17 @@ func (c *EngineConfig) validatePaths() error {
 	return nil
 }
 
+func (c *EngineConfig) validateRuntimeNames() error {
```

Comment:
you need to create a an empty stub in config_remote.go to make it compile in remote mode.